build/patches/Allow-building-without-enable_reporting.patch

```diff
@@ -73,6 +80,130 @@ diff --git a/content/browser/devtools/protocol/network_handler.h b/content/brows
  
    // TODO(dgozman): Remove this.
    const std::string host_id_;
+diff --git a/content/browser/net/cross_origin_embedder_policy_reporter.cc b/content/browser/net/cross_origin_embedder_policy_reporter.cc
+--- a/content/browser/net/cross_origin_embedder_policy_reporter.cc
++++ b/content/browser/net/cross_origin_embedder_policy_reporter.cc
+@@ -107,6 +107,7 @@ void CrossOriginEmbedderPolicyReporter::QueueAndNotify(
+         kType, context_url_, blink::mojom::ReportBody::New(std::move(list))));
+   }
+   if (endpoint) {
++#if BUILDFLAG(ENABLE_REPORTING)
+     base::DictionaryValue body_to_pass;
+     for (const auto& pair : body) {
+       body_to_pass.SetString(pair.first, pair.second);
+@@ -117,6 +118,7 @@ void CrossOriginEmbedderPolicyReporter::QueueAndNotify(
+         kType, *endpoint, context_url_, reporting_source_,
+         network_isolation_key_,
+         /*user_agent=*/absl::nullopt, std::move(body_to_pass));
++#endif
+   }
+ }
+ 
+diff --git a/content/browser/net/cross_origin_opener_policy_reporter.cc b/content/browser/net/cross_origin_opener_policy_reporter.cc
+--- a/content/browser/net/cross_origin_opener_policy_reporter.cc
++++ b/content/browser/net/cross_origin_opener_policy_reporter.cc
+@@ -16,11 +16,14 @@
+ #include "services/network/public/mojom/network_context.mojom.h"
+ #include "services/network/public/mojom/source_location.mojom.h"
+ #include "url/origin.h"
++#include "build/build_config.h"
+ 
+ namespace content {
+ 
+ namespace {
+ 
++#pragma GCC diagnostic push
++#pragma GCC diagnostic ignored "-Wunused-const-variable"
```

Comment:
Why not remove the constants instead? This warning ignore casts a doubt on whether something is used or not.

Reply:
yes, in fact maybe it's better, I don't remember why I preferred that solution.

Reply:
fixed in https://github.com/bromite/bromite/pull/1556/commits/88c7018cb0e80ed53bff52aa2d3993d00c66acd4